### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

**Review by normalhuman — APPROVED** [review]
Indeed, these all are typos.

File: sympy/simplify/hyperexpand.py [hunk 1]
```diff
@@ -942,7 +942,7 @@ class Operator(object):
     *not* blindly differentiate but instead use a different representation
     of the z*d/dz operator (see make_derivative_operator).
 
-    To subclass from this, define a __init__ method that initalises a
```
Comment (ylemkimon) [inline sympy/simplify/hyperexpand.py]: initializes

Comment (normalhuman) [reply sympy/simplify/hyperexpand.py]: Both are correct, it's American vs British spelling. ([Example](https://askubuntu.com/q/195073)). I would go with the preference of the original author of the text.

Comment (ylemkimon) [reply sympy/simplify/hyperexpand.py]: @normalhuman But SymPy's documentation is mostly written in US English:
```
C:\Users\wisec\Documents\GitHub\sympy>git grep initialise | wc -l
2

C:\Users\wisec\Documents\GitHub\sympy>git grep initialize | wc -l
112
```
I think it's better to be consistent through the documentation.

Comment (luzpaz) [reply sympy/simplify/hyperexpand.py]: You'll also need to check initialis{es|ation|ing}

Comment (ylemkimon) [reply sympy/simplify/hyperexpand.py]:
```
C:\Users\wisec\Documents\GitHub\sympy>git grep initialis | wc -l
3

C:\Users\wisec\Documents\GitHub\sympy>git grep initializ | wc -l
143
```

Comment (luzpaz) [reply sympy/simplify/hyperexpand.py]: I've made the change to American spelling. Shall I rebase and force push or just push for you guys to review then i or you can squash the commits ?

Comment (smichr) [reply sympy/simplify/hyperexpand.py]: Having just reviewed these changes, I'm going to commit them so the next person doesn't have to look again. If you make a new PR with the americanized spellings that would be fine.

**Pull request merged**